pkg/api/daemon_example.json

```diff
@@ -0,0 +1,78 @@
+{
+    "kind": "Daemon",
```

Comment:
node_example.json, pod_example.json, etc are all in pkg/api, so I thought I should put daemon_example.json in the same place. It's used primarily for testing in pkg/api/conversion_test.go : BenchmarkDaemonConversion. Should I move the file elsewhere, and change the path in the test? Thanks!